app/views/works/new_submission.html.erb

```diff
@@ -21,12 +13,12 @@
   <p>If you have already started a submission for this deposit, then please go back
   to the home page and complete the existing submission. If you would like to talk to a
   specialist about your submission, we may be reached at
-  <a href="mailto:1-800-go-hannah@princeton.edu">1-800-go-hannah@princeton.edu</a>
+  <a href="mailto:prds@princeton.edu">prds@princeton.edu</a>
```

Comment:
😁  I think I like the old contact info better

Reply:
Me too honestly :D